### PR TITLE
Add service to OVA for default VCH install wizard

### PR DESCRIPTION
**Pull request by andrewtchin**
Partial for 4731

Install systemd unit for OVA engine installer, update makefiles, add VIC engine install wizard section to OVA deploy

Following work will add a web app for users to input values that will be used to run `vic-machine create` to make a default VCH

### CODE REVIEW

File: installer/packer/vic-unified.ovf [hunk 1]
```diff
@@ -294,6 +294,38 @@ EVALUATION LICENSE. If You are licensing the Software for evaluation purposes, Y
         <Description>Paste in the content of certificate key file. Leave blank for a generated key.</Description>
       </Property>
     </ProductSection>
+    <ProductSection ovf:class="engine_installer" ovf:required="false">
+      <Info>VIC Engine Install Wizard Properties</Info>
+      <Category>6. VIC Engine Install Wizard Configuration</Category>
+      <Property ovf:key="deploy" ovf:type="boolean" ovf:userConfigurable="true" ovf:value="false">
+        <Label>6.1. Enable VIC Engine Install Wizard</Label>
+        <Description>When setting this to true, starts the VIC Engine Install Wizard web application.</Description>
```
Comment (frapposelli) [inline installer/packer/vic-unified.ovf]: Do we really want to make this optional?

File: installer/packer/vic-unified.ovf [hunk 2]
```diff
@@ -294,6 +294,38 @@ EVALUATION LICENSE. If You are licensing the Software for evaluation purposes, Y
...
+      </Property>
+      <Property ovf:key="vcenter_addr" ovf:qualifiers="MinLen(0),MaxLen(65535)" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
+        <Label>6.2. vCenter Address</Label>
+        <Description>IP address or FQDN of the vCenter. Leave blank if not using VIC Engine Install Wizard.</Description>
+      </Property>
+      <Property ovf:key="admin_user" ovf:qualifiers="MinLen(0),MaxLen(65535)" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
+        <Label>6.3. vCenter Admin Username</Label>
+        <Description>Username of the vCenter admin user. Leave blank if not using VIC Engine Install Wizard.</Description>
+      </Property>
+      <Property ovf:key="admin_password" ovf:password="true" ovf:qualifiers="MinLen(0),MaxLen(65535)" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
```
Comment (frapposelli) [inline installer/packer/vic-unified.ovf]: We shouldn't be asking for vcenter credentials here, we should ask them in the engine installer.

File: installer/packer/vic-unified.ovf [hunk 3]
```diff
@@ -294,6 +294,38 @@ EVALUATION LICENSE. If You are licensing the Software for evaluation purposes, Y
...
+        <Label>6.4. vCenter Admin Password</Label>
+        <Description>Password of the vCenter admin user. Leave blank if not using VIC Engine Install Wizard.</Description>
+      </Property>
+      <Property ovf:key="port" ovf:qualifiers="MinValue(1),MaxValue(65535)" ovf:type="int" ovf:userConfigurable="true" ovf:value="1337">
+        <Label>6.5. Engine Install Wizard Port</Label>
+        <Description>Specifies the port on which fileserver will be published.</Description>
+      </Property>
+      <Property ovf:key="ssl_cert" ovf:qualifiers="MinLen(0),MaxLen(65535)" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
```
Comment (frapposelli) [inline installer/packer/vic-unified.ovf]: I believe this is fine with a self signed cert, it's a one-time installer.

**Comment by frapposelli** [comment]
LGTM

File: installer/engine_installer/engine_installer.go [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("hello world")
```
Comment (gigawhitlocks) [inline installer/engine_installer/engine_installer.go]: oh hai der

Comment (gigawhitlocks) [inline installer/engine_installer/engine_installer.go]: is me, world!

Comment (andrewtchin) [inline installer/engine_installer/engine_installer.go]: HAI

**Pull request merged**